.devcontainer/Dockerfile

```diff
@@ -19,5 +19,5 @@ RUN yarn -v
 # This is so that when someone updates their devcontainer name, it doesn't get screwed up
 RUN echo 'first_workspace=\"$(cd /workspaces && ls | head -1)\"; alias spellcheck="cspell --config /workspaces/$first_workspace/cSpell.json \"/workspaces/$first_workspace/docs/**/*.md\" --no-progress"' >> ~/.bashrc
 RUN echo 'first_workspace=\"$(cd /workspaces && ls | head -1)\"; alias lint="markdownlint -c /workspaces/$first_workspace/.github/linters/.markdown-lint.yml /workspaces/$first_workspace/docs/**/*.md"' >> ~/.bashrc
-RUN echo 'first_workspace=\"$(cd /workspaces && ls | head -1)\"; alias build="cd /workspaces/$first_workspace/docs && docfx build"' >> ~/.bashrc
-RUN echo 'first_workspace=\"$(cd /workspaces && ls | head -1)\"; alias serve="build && docfx serve _site -n \"*\""' >> ~/.bashrc
+RUN echo 'first_workspace=\"$(cd /workspaces && ls | head -1)\"; alias build="docfx /workspaces/$first_workspace/docs/docfx.json"' >> ~/.bashrc
```

Comment:
ℹ️ Incidentally I realized our dev container syntax could be cleaner so I'm tacking it on here. Verified in my local devcontainer.